src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx

```diff
@@ -355,6 +368,9 @@ export const K8SResourceList = ({
                 renderCallBackSync={renderCallBackSync}
             />
             {resourceListLoader ? <Progressing pageLoader /> : renderList()}
+            {PodRestart && (
+                <PodRestart clusterId={clusterId} name={name ?? 'demo2-vishu-env-5ddcb8f64b-lgrqk'} namespace={namespace ?? 'non-prod'} />
```

Comment:
pls revert when testing is done

Reply:
also pls use useParams inside the components

Reply:
Done